src/components/common/FocusTrapper.ts

```diff
@@ -1,36 +0,0 @@
-import _ from 'lodash/fp';
-import FocusLock from 'react-focus-lock';
-import { h } from 'react-hyperscript-helpers';
-
-// react-focus-lock does not export a type for FocusLock's props, but since FocusLock
-// is a function component, we can get the type from its parameters.
-type FocusLockProps = Parameters<typeof FocusLock>[0];
-
-export interface FocusTrapperProps extends FocusLockProps {
```

Comment:
This was wrong. FocusTrapper doesn't pass on props to the underlying FocusLock, it passes them to its `lockProps`, where they're passed to the rendered div element.